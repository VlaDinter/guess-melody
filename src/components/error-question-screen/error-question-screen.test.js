import React from 'react';
import renderer from 'react-test-renderer';

import ErrorQuestionScreen from './error-question-screen.jsx';

it(`ErrorQuestionScreen is rendered correctly`, () => {
  const tree = renderer.create(<ErrorQuestionScreen
    mistakes={0}
    maxMistakes={0}
  />).toJSON();

  expect(tree).toMatchSnapshot();
});
