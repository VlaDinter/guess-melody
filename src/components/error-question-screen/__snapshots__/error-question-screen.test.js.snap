// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorQuestionScreen is rendered correctly 1`] = `
<div
  className="game__mistakes"
/>
`;
